frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/mention/mention_page_block.dart

```diff
@@ -399,7 +399,7 @@ Future<void> _handleDoubleTap(
         {
           MentionBlockKeys.mention: {
             MentionBlockKeys.type: MentionType.page.name,
-            MentionBlockKeys.pageId: newViewId,
+            MentionBlockKeys.pageId: newView.id,
```

Comment:
Could we add a type check to the mention block to prevent potential misuse elsewhere?

Reply:
Done!